### PR TITLE
feat: fix env vars + update site metadata

### PR DESCRIPTION
**Pull request by parketh**
## Summary

This PR: 
1. fixes env vars to be ingested at runtime by building the Docker image at startup (similar to `op-bridge-ui` approach)
2. updates the site metadata
3. applies Snapchain branding

## Test plan

```
yarn dev
```
Right click, click `Inspect` -> `Elements` tab -> check metadata included under `<Header>` tab

For live deployments, the social media preview should look like this:

image

## Post-deployment

We need to update `babylon-deployment` by pulling the latest Docker image, and rebuild this site.

### CODE REVIEW

**Review by lesterli — APPROVED** [review]

File: src/config/index.ts
Comment (bap2pecs) [inline src/config/index.ts:1]: why are all the tests removed?

File: Makefile [hunk 1]
```diff
@@ -1,13 +0,0 @@
-DOCKER ?= $(shell which docker)
-GIT_ROOT := $(shell git rev-parse --show-toplevel)
-
-build: npm run build
-
-build-docker:
```
Comment (bap2pecs) [inline Makefile:6]: I think we should still have `build-docker`

**Review by bap2pecs — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -13,40 +13,13 @@ COPY next.config.mjs .
...
-# Replace the variables in the .env file
-ARG NEXT_PUBLIC_FINALITY_GADGET_API_URL
-ARG NEXT_PUBLIC_DISPLAY_TESTING_MESSAGES
-RUN sed -i "s|NEXT_PUBLIC_FINALITY_GADGET_API_URL=.*|NEXT_PUBLIC_FINALITY_GADGET_API_URL=${NEXT_PUBLIC_FINALITY_GADGET_API_URL}|g" .env
-RUN sed -i "s|NEXT_PUBLIC_DISPLAY_TESTING_MESSAGES=.*|NEXT_PUBLIC_DISPLAY_TESTING_MESSAGES=${NEXT_PUBLIC_DISPLAY_TESTING_MESSAGES}|g" .env
+COPY package.json .
+COPY package-lock.json .
 
-RUN npm run build
+RUN npm install
```
Comment (bap2pecs) [inline Dockerfile:19]: i don't think we need these 3 lines. they are duplicate with

```
COPY package.json package-lock.json ./
# Install dependencies with npm
RUN npm ci
```

above

**Comment by bap2pecs** [comment]
now it looks like this:

image

File: src/config/index.ts
Comment (parketh) [reply src/config/index.ts:1]: this is legacy code copied from babylon's staking repo and not used in our code, so cleaned it up

**Pull request merged**